Frends.RabbitMQ.Publish/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.1.1] - 2023-03-14
```

Comment:
You have added a new parameter, so you should increase minor version by one instead of build version.